simphony/cuds/abc_modeling_engine.py

```diff
@@ -37,16 +37,9 @@ def add_lattice(self, lattice):
 
         Parameters
         ----------
-        lattice: ABCLattice
+        lattice : ABCLattice
             lattice to be added.
 
-        Returns
```

Comment:
I thought that this comment was specific to the CudsFile object and not to the generic engine 
adapter.

- According to the wiki discussion when I add a CUDS container into an engine only the compatible data are transferred to the internal engine. That means that when the a data container with the `name` is requested from an engine it is not guarantied that it will contain all the original information. Having the add_XXX methods return a proxy object could confuse since in-general you would not expect that an add method would return a similar object back.

```python
mesh = Mesh('my_mesh')
engine_mesh = engine.add_mesh(mesh)
```
instead of the more specific

```python
mesh = Mesh('my_mesh')
engine.add_mesh(mesh)
engine_mesh = engine.get_mesh('My mesh')
```

However, I can see that this should probably go through some discussion. So I will revert the changes and add a question issue